common/_v1alpha1_kruize.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: my.domain/v1alpha1
```

Comment:
Is this being used?

Reply:
@ncau if this file is not being used, pelase remove it from the PR